modules/standard/Random.chpl

```diff
@@ -769,8 +768,7 @@ module Random {
     use super.RandomSupport;
     private use Random;
     public use PCGRandomLib;
```

Comment:
It seems to me that PCGRandomLib should be private used and then people should import/use it as a submodule if they need it.

Reply:
I admittedly focused fairly exclusively on the public uses in internal modules in this effort (other than changes required to standard modules to keep tests working), just to limit the scope and because I didn't want to make this a review of what each standard module did or didn't make available by default.  It looks like this has been a public use since you added it, though it didn't get explicitly marked as public until we made `use` private by default for standard modules.  Would it be OK to leave that for a future/separate effort?  Or do you feel pretty confident that it shouldn't break anything?

Reply:
Future / separate effort would be fine. I'm also confident that it shouldn't break anything to severely (it's really esoteric and I really doubt that anybody is using it; but if there is 1 person who is, I'm comfortable asking them to add a use/import of the submodule).

Reply:
Testing library/standard/Random with that change, I see failures in:

```
[Error matching compiler output for library/standard/Random/ferguson/pcg-rand-check]
[Error matching compiler output for library/standard/Random/ferguson/pcg-rand-range-check]
[Error matching compiler output for library/standard/Random/ferguson/pcg-rand-range-check2]
```

which is maybe not surprising given their nature?  If you agree, I can add the uses to these tests and do a more complete test.

Reply:
I've started on this in https://github.com/chapel-lang/chapel/pull/16127